### ISSUE TITLE
[Bug][a11y]: Slider editable NumberField labeled twice

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3322]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Slider, FieldLabel, NumberField

### Expected behavior

The NumberField that displays in the shadowDom for the `editable` variant of a Slider should only include the `textContent` for its `label` once as its `aria-label`.

A possible solution for this would be to deduplicate within the `get labelText()` method in FieldLabel, as follows:

```ts
    private get labelText(): string | undefined {
        const assignedNodes = this.slotEl.assignedNodes({ flatten: true });
        if (!assignedNodes.length) {
            return undefined;
        }
        const labelText = assignedNodes.map((node) =>
            (node.textContent || /* c8 ignore next */ '').trim()
        )
        /** deduplicate and omit empty nodes */
        .filter(
            (textContent, index, array) => (
                textContent.length && 
                array.findIndex((item) => item === textContent) === index
            )
        );
        return labelText.join(' ');
    }
```

### Actual behavior

On https://opensource.adobe.com/spectrum-web-components/components/slider/#editable, the NumberField that displays in the shadowDom for the `editable` variant of a Slider repeats the `textContent` for its `label` twice in its `aria-label`. 

Rendered `<sp-number-field>` element in shadowDom, where you can see `aria-label=" Angle (editable)  Angle (editable) "` on the `input`, follows:
```html
<sp-number-field id="number-field" min="0" max="360" step="1" value="180" hide-stepper="" dir="ltr" type="text" step-modifier="10" autocomplete="off" focusable="" data-dashlane-shadowhost="true" data-dashlane-observed="true">
    <!-- #shadow-root -->
      <div id="textfield">
          <!--?lit$812536239$-->
          <!--?lit$812536239$-->
          <!--?lit$812536239$-->
          <!--?lit$812536239$-->
          <!-- @ts-ignore -->
          <input class="input" type="text" aria-describedby="sp-help-text-1" aria-label=" Angle (editable)  Angle (editable) " placeholder="" autocomplete="off" inputmode="numeric" data-dashlane-rid="cde23094b63176a0" data-form-type="other">
          <!--?lit$812536239$--><!--?-->
      </div>
</sp-number-field>
```

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by susmithayenugula** [comment]
Hi team,
Took a look at this and I am unable to reproduce this issue. 

<img width="1439" alt="Screenshot 2024-03-05 at 1 05 50 AM" src="https://github.com/adobe/spectrum-web-components/assets/21351513/58c73aa3-5e24-4990-b409-50df2cf25ea4">
<img width="1439" alt="Screenshot 2024-03-05 at 1 03 52 AM" src="https://github.com/adobe/spectrum-web-components/assets/21351513/59a4eb4f-d93e-495e-bd2b-b5d322f70dd7">

**Comment by Westbrook** [comment]
Issues that were accidentally fixed over time, are some of the best ones!! @majornista unless you're seeing something different, we'd be quite happy to close this one due to 🪄...

**Comment by Westbrook** [comment]
Closing as no longer repro-able. If this issue arises again, feel free to reopen with additional reproduction steps.
